gcsio/src/main/java/com/google/cloud/hadoop/gcsio/authorization/GcsResourceAndActions.java

```diff
@@ -0,0 +1,51 @@
+package com.google.cloud.hadoop.gcsio.authorization;
+
+import java.util.Objects;
+
+public class GcsResourceAndActions {
```

Comment:
Looks like the name of class should be "GcsResourceAndAction"? with "action" being singular

Reply:
Yep, I missed the class name.